projects/example-app-cypress/support/index.js

```diff
@@ -0,0 +1,3 @@
+import '@applitools/eyes.cypress/commands'
+
```

Comment:
- End with a semicolon
- Remove extra line break

Reply:
* semicolon added
*extra line break removed
*eyes.cypress is now eyes-cypress